Data/Scripts/001_Technical/002_RubyUtilities.rb

```diff
@@ -50,6 +50,10 @@ def cut(bitmap, width)
     end
     return new_string
   end
+
+  def is_numeric?
+    self.scan(/[^0-9\.\-]/).count < 1
```

Comment:
This would return true if you had "007.42.33.9" or "42-...6---88", which aren't really numerics. Would `return self == self.to_i.to_s` work? It still wouldn't catch excessive leading zeroes, but it's a start.

Also, let's have explicit `return`s.

Reply:
Hmm, you're right. I'll whip up another regex to only work on `x.x` and `-x.x` format.

`self == self.to_i.to_s` would work only for integers. So not ideal in a global case.